pkg/util/refreshable/refreshable.go

```diff
@@ -59,8 +59,7 @@ func (a *authorizer) RefreshWithContext(ctx context.Context, log *logrus.Entry)
 		log.Info(err)
 
 		if !autorest.IsTokenRefreshError(err) ||
-			azureerrors.IsUnauthorizedClientError(err) ||
-			azureerrors.IsInvalidSecretError(err) {
```

Comment:
Do we need to update the docstring for this function?

Reply:
Also - we rely on this function in the following places:

* `pkg/util/aad/aad.go`
* `pkg/util/steps/refreshing.go`

This PR updates the first one. Asuming that the changes in `pkg/util/aad/aad.go` are correct, it looks like we are likely breaking `pkg/util/steps/refreshing.go` and everything which depends on it.